### PR TITLE
Add Ruby-GCCcore-10.3.0.eb to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Trying to add Ruby-GCCcore-10.3.0.eb to NESSI/2022.11

### CODE REVIEW

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -424,7 +424,7 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
 #$EB Perl-5.32.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2640
 #use enhanced CMake easyblock to patch CMake's UnixPaths.cmake script if --sysroot is set
 #from https://github.com/easybuilders/easybuild-easyblocks/pull/2248
-$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
+#$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:427]: Lets keep it uncommented for the moment.

File: EESSI-pilot-install-software.sh [hunk 2]
```diff
@@ -449,6 +449,7 @@ $EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
 #####################
 ### add packages here
 #####################
+$EB Ruby-3.0.1-GCCcore-10.3.0.eb --robot
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:452]: Just curious, why was Ruby considered?

File: eessi-2022.11.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 easyconfigs:
   - EasyBuild-4.7.0.eb
-  - CMake-3.20.1-GCCcore-10.3.0.eb
```
Comment (trz42) [inline eessi-2022.11.yml:3]: I'd leave CMake in the easy stack file.

**Review by trz42 — CHANGES_REQUESTED** [review]
Left a few comments.

File: eessi-2022.11.yml (hunk 3 above)
Comment (TopRichard) [reply eessi-2022.11.yml:3]: Done

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: Done

File: EESSI-pilot-install-software.sh (hunk 2 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh:452]: No specific reason

**Comment by trz42** [comment]
Builds for Fram are missing in the last batch. We may want to retry this later on.

**Comment by trz42** [comment]
@TopRichard could you investigate why the builds on Fram failed? Maybe a temporary issue. Might be ok/easiest, to just retrigger the builds and only investigate if the problem persists.

**Pull request closed without merging**